components/phaser/gura3mil/main.tsx

```diff
@@ -99,7 +99,15 @@ class Main extends Phaser.Scene {
 			.setAlpha(0)
 			.on('scroll', ({ t }: { t: number }) => {
 				if (t >= 0.8) {
-					if (submissions.length <= 0) return this.panel.removeAllListeners('scroll');
+					if (submissions.length <= 0) {
```

Comment:
Don't get why it's `<= 0` instead of `=== 0` since it's never below 0, but whatever.

Reply:
I did it to be "safe" but yeah